SuiteCareers/SuiteCareers/Models/User.cs

```diff
@@ -3,21 +3,25 @@
 
 namespace SuiteCareers.Models
 {
-    
-        public class User
-        {
-            public long Id { get; set; }
-            [DataType(DataType.EmailAddress)]
-            public string? Email { get; set; }
-            
-            public string? FirstName { get; set; }
-            
-            public string? LastName { get; set; }
-            
-            public string? City { get; set; }
-            
-            public string? State { get; set; }
-        }
+
+    public class User
+    {
+        [Key]
+        public long UserId { get; set; }
+        [DataType(DataType.EmailAddress)]
+        public string? Email { get; set; }
+        [Required(ErrorMessage = "Please enter your first name")]
+        public string? FirstName { get; set; }
+        [Required(ErrorMessage = "Please enter your last name")]
+        public string? LastName { get; set; }
+        [Required(ErrorMessage = "Please enter the city of which you live")]
+        public string? City { get; set; }
+        [Required(ErrorMessage = "Please Enter the state of which you live")]
```

Comment:
These error message don't make any sense in the Entity model, only in a ViewModel. 
Just use the `[Required]` notation to avoid confusion